src/pip/_internal/models/wheel.py

```diff
@@ -58,7 +58,10 @@ def support_index_min(self, tags: List[Tag]) -> int:
         :raises ValueError: If none of the wheel's file tags match one of
             the supported tags.
         """
-        return min(tags.index(tag) for tag in self.file_tags if tag in tags)
+        try:
+            return next(i for i, t in enumerate(tags) if t in self.file_tags)
+        except StopIteration:
+            raise ValueError()
```

Comment:
If you want to raise a custom exception, it’d be better to provide a meaningful error message.

Reply:
I don't care about the exception, but existing tests do, and it seems preferable to preserve backwards compatibility since this specific exception class is mentioned in the documentation of this method...